src/main/java/org/datadog/jenkins/plugins/datadog/DatadogGlobalConfiguration.java

```diff
@@ -290,7 +292,9 @@ public boolean configure(final StaplerRequest req, final JSONObject formData) th
                     this.getTargetApiURL(), this.getTargetApiKey(), this.getTargetHost(), this.getTargetPort());
 
         } catch(Exception e){
-            logger.warning("Unexpected exception occurred - " + e.getMessage());
+            StringWriter sw = new StringWriter();
```

Comment:
maybe it worth it to wrap it to a custom logger function.

Reply:
I thought about it but i am not convinced it will be less lines of code overall. I change it tho.